---
title: COVID Wastewater Scotland
---
{% include navbar.html %} 

# Summary of methodology development

This project used three approaches for detection and quantification of new SARS-COV2 variants in wastewater: RT-qPCR, droplet digital PCR (ddPCR) and next generation sequencing (NGS). 

## RT-qPCR

The first technique tried was RT-qPCR. Several tests were performed, from choosing the best kits for RT-PCR to testing different sets of primers and probes to check if they were able to amplify SARS2 from wastewater samples. 
More specifically, the team focused on detecting new variants by targeting mutations that can discriminate between the different SARS2 lineages, which are often found in the S-gene.
To test the specificity of variant detection, primers and probes were optimised using synthetic RNA templates that were a mix of WT and mutant sequences.
After several attempts of quantifying the SARS2 variants using RT-qPCR with different sets of primers and probes, the group found that the probes developed could discriminate between variants in synthetic samples, but they could not measure the levels of different variants in wastewater with sufficient level of certainty. They concluded that because wastewater samples contain a mix of variants that are all themselves mutating, different primer/probe combinations struggle to discriminate between them, making this type of sample more challenging to analyse. 

## ddPCR (Digital Droplet PCR)

The group then followed on to try a novel technique called droplet digital PCR (also known as Crystal digital PCR) to detect variants. This technique had the potential to be more successful to detect viral RNA as it has been demonstrated that this is a more sensitive technique for detecting mutations in target sequences.
ddPCR worked well for discriminating between synthetic viral variants that were present at high concentrations in samples, but the team consistently found that ddPCR was less sensitive than RT-qPCR. 

## NGS (Next Generation Sequencing)

In parallel to using qPCR and digital PCR, the team have developed NGS methods for analysing SARS2 variants in wastewater. The advantages of this approach are that it is unnecessary to make a priori assumptions about the presence of mutations and that new variants can be identified in samples de novo. Little material is required for characterising the spectrum of variants. However, there are also disadvantages. The laboratory methods are challenging and time consuming. It takes at least 4 days for results to be generated from a wastewater sample, and then the information needs to be analysed by a bioinformatics team. However, the information provided is extremely rich and can be analysed in multiple different ways.
Since optimising the NGS methodology, the research team has analysed over 2000 wastewater samples from across Scotland and with the support of Biomathematics & Statistics Scotland (BioSS) this information has been used to monitor the spread of the omicron variant in some of the  wastewater treatment works. 

![image](https://user-images.githubusercontent.com/29427778/160816030-8c869aee-9bc7-4112-aac6-3bdbf6891bdf.png)

Source: Coronavirus (Covid-19): modelling the epidemic in Scotland (Issue No. 92)



# Open Notebooks

We share some of the Electronic Laboratory Notebooks (ELNs) used by Gilbert lab researchers, on the Benchling platform (https://www.benchling.com), which is popular among molecular biologists. The notebooks document the research processes behind the new methodology. In our experience the records are exemplary with high educational value, as well as constituting evidence of the professionalism and scientific rigor of Professor Nick Gilbertâ€™s group.

* [Obtaining standard curves for L452R, K417T and E484K strains](https://benchling.com/s/etr-yfHurJIagMvvI65CXT32?m=slm-KAATs3hjtXDVOYbrMGtc)
* [Running RT-qPCR on undiluted/unconcentrated COVID wastewater RNA samples](https://benchling.com/s/etr-vxcvSCsOTJLJ5Uj3sy1i?m=slm-QKaMgfEomMVCDtQfIwJW)
* [Testing different qPCR mixes on crystal PCR](https://benchling.com/s/etr-8u10BD9KalyiIh9ASHXA?m=slm-IOyleZlGh3xMpJXoWRrO)
* [Test of NGS Library Prep on successfully amplified cDNA from wastewater](https://benchling.com/s/etr-Cj7F4V7aeiOidiV8ZQzI?m=slm-4X00YePdQLjw8oR2kQDE)
* [Attempting ARTIC SARS-CoV-2 cDNA amplification on concentrated wastewater](https://benchling.com/s/etr-0FdV1H0rpWeHk4H72NOg?m=slm-WgoF2PnZrLLK2lE5gfb1)


{% include bottomnavbar.html %}
