---
title: COVID Wastewater Scotland
---
{% include navbar.html %} 

# SARS-CoV-2 Variants Monitoring in Wastewater

The SARS-COV-2 virus (SARS2), the causative agent for COVID-19, mutates as it adapts to different evolutionary pressures. 
From the analysis undertaken so far it appears that new variants are becoming more infective, but fortunately there is 
similar disease severity. However, it is difficult to predict what new viruses will emerge and critical for this project 
it is important to know when they first appear in Scotland and how they then spread. The goal of this project was not to 
identify new variants. The UK, through the COG consortium, are leading efforts to identify new Variants of Concern (VoC). 
Instead, this project aimed to develop methods for identifying these new variants to provide real-time data to Public Health 
Scotland and related agencies to monitor and respond to new variants. 
In addition to identifying new variants this project attempted to quantify the amount of SARS2 in wastewater compared to human markers, for example crAssphage. 
The research team with the expertise to deliver the ‘Variants’ component of this project are based at the MRC Human Genetics Unit, 
University of Edinburgh (UOE) and have broad experience in molecular biology techniques, including next generation 
sequencing. Importantly, the ‘Variants’ team started screening for SARS2 as a node for NHS Lothian in Summer 2020, and 
since have developed and implemented methodologies for screening saliva for SARS2 as part of a project called TestEd which 
is supported by a £1.8M UKRI grant. TestEd both screens for SARS2 but has also developed PCR assays to identify alpha, 
beta, gamma and delta variants.


{% include footer.html %} 

# Summary of methodology development

This project used three approaches for detection of variants, RT-qPCR, droplet digital PCR (ddPCR) and next generation sequencing (NGS). 

RT-qPCR

The first technique tried was RT-qPCR. Several tests were performed, from testing best kits for RT-PCR to testing different sets of primers and probes to check if they were able to amplify SARS-CoV-2 from wastewater samples. 
More specifically, the team focused on detecting new variants by targeting mutations that can discriminate between the different SARS-CoV-2 lineages, which are often found in the S-gene.
To test the specificity of variant detection, primers and probes were optimised using synthetic RNA templates that were a mix of WT and mutant sequences.
After several attempts of quantifying the SARS-CoV-2 variants using RT-qPCR with different sets of primers and probes, the group found that the probes developed could discriminate between variants in synthetic samples, but they could not measure the levels of different variants in wastewater with sufficient level of certainty. They concluded that because wastewater samples contain a mix of variants that are all themselves mutating, different primer/probe combinations struggle to discriminate between them, making this type of sample more challenging to analyse. 

ddPCR (Digital Droplet PCR)

The group then followed on to try a novel technique called droplet digital PCR (also known as Crystal digital PCR) to detect variants. This technique had the potential to be more successful to detect viral RNA as it has been demonstrated that this is a more sensitive technique for detecting mutations in target sequences.
ddPCR worked well for discriminating between synthetic viral variants that were present at high concentrations in samples, but the team consistently found that ddPCR was less sensitive than RT-qPCR. 

NGS (Next Generation Sequencing)

In parallel to using qPCR and digital PCR, the team have developed NGS methods for analysing SARS-CoV-2 variants in wastewater. The advantages of this approach are that it is unnecessary to make a priori assumptions about the presence of mutations and that new variants can be identified in samples de novo. Little material is required for characterising the spectrum of variants. However, there are also disadvantages. The laboratory methods are challenging and time consuming. It takes at least 4 days for results to be generated from a wastewater sample, and then the information needs to be analysed by a bioinformatics team. However, the information provided is extremely rich and can be analysed in multiple different ways.
Since optimising the NGS methodology, the research team has analysed over 2000 wastewater samples from across Scotland and with the support of Biomathematics & Statistics Scotland (BioSS) this information has been used to monitor the spread of the omicron variant in some of the  wastewater treatment works. 


